improver/calibration/ensemble_calibration.py

```diff
@@ -1366,6 +1367,61 @@ def process(
         return coefficients_cubelist
 
 
+class MetaEstimateCoefficientsForEnsembleCalibration(BasePlugin):
+    """
+    Meta plugin for handling directories of netcdfs as inputs, instead of cubes
+    """
+
+    def __init__(
+        self,
+        distribution,
+        truth_attribute,
+        cycle_point=None,
+        max_days_offset=None,
+        point_by_point=False,
+        use_default_initial_guess=False,
+        units=None,
+        predictor="mean",
+        tolerance: float = 0.02,
+        max_iterations: int = 1000,
+    ):
+        self.distribution = distribution
+        self.truth_attribute = truth_attribute
+        self.cycle_point = cycle_point
+        self.max_days_offset = max_days_offset
+        self.point_by_point = point_by_point
+        self.use_default_initial_guess = use_default_initial_guess
+        self.units = units
+        self.predictor = predictor
+        self.tolerance = tolerance
+        self.max_iterations = max_iterations
```

Comment:
it's good form to suffix these with an underscore to indicate 'private' (considered private by convention, not in the same sense of private in c/c++ of course).
Basically, all data should be hidden within the class.  If we really want to expose this data to users, then they should be exposed via properties (setters & getters).  This stops users from easily tweaking values they aren't intended to be tweaking.